docs/docs/attestation/sbom.md

```diff
@@ -0,0 +1,44 @@
+# SBOM attestation
+
+[Cosign](https://github.com/sigstore/cosign) supports generating and verifying [in-toto attestations](https://github.com/in-toto/attestation). This tool enables you to sign and verify SBOM attestation.
+
+## Sign with a local key pair
+
+In the following example, Trivy generates an SBOM in the spdx format, and then Cosign attaches an attestation of the SBOM to a container image with a local key pair.
+
+```
+$ trivy image --format spdx -o predicate <IMAGE>
+$ cosign attest --key /path/to/cosign.key --type spdx --predicate predicate <IMAGE>
+```
+
+Then, you can verify attestations on the image.
+
+```
+$ cosign verify-attestation --key /path/to/cosign.pub <IMAGE>
+```
+
+You can also create attestations of other formatted SBOM.
+
+```
+# spdx-json
+$ trivy image --format spdx-json -o predicate <IMAGE>
+$ cosign attest --key /path/to/cosign.key --type spdx --predicate predicate <IMAGE>
+
+# cyclonedx
+$ trivy image --format cyclonedx -o predicate <IMAGE>
+$ cosign attest --key /path/to/cosign.key --type https://cyclonedx.org/schema --predicate predicate <IMAGE>
```

Comment:
The cyclonedx type is available in [the main branch](https://github.com/sigstore/cosign/blob/8f2c36dc478904859d41953adba08a8555b94b46/doc/cosign_attest.md?plain=1#L66). But the current latest release(v1.9.0) does not support it yet.
So [the cyclonedx predicate type](https://github.com/in-toto/in-toto-golang/blob/fa494aaa0add184054d127bb3fa25ca260723551/in_toto/model.go#L83
) is specified directly.